src/Components/v2/Carousel.tsx

```diff
@@ -230,17 +234,21 @@ export class BaseCarousel extends React.Component<
   }
 
   renderLeftArrow = () => {
-    const { onArrowClick, renderLeftArrow, showArrows, height } = this.props
+    const {
+      arrowPosition,
+      onArrowClick,
+      renderLeftArrow,
+      showArrows,
+      height,
+    } = this.props
 
     if (!showArrows) {
       return null
     }
 
     const leftPosition =
-      this.props.arrowPosition !== null &&
-      this.props.arrowPosition !== undefined
-        ? this.props.arrowPosition
-        : -space(4)
+      arrowPosition != null ? this.props.arrowPosition : -space(4)
```

Comment:
I am curious about why did you make this change here? The reason I ask this is the undefined checking seems to make sense. 
Also, you already did the destructing above. When the condiction was true, is there any differences between this.props.arrowPosition and arrowPosition?

Reply:
So, the change with `!== undefined` and `!== `null` to `!=` is a JS shortcut for checking both undefined _and_ null at the same time. Its a weird hack :) 

Re the last `this.props.arrowPosition`, oops! I need to update that spot too. Nice catch 👍